src/content/api/cli.md

```diff
@@ -134,55 +134,73 @@ webpack --env.platform=web  # sets env.platform == "web"
-`--build-delimiter` | Display custom text after build output | string | Default string is null. You could provide a string such as `=== Build done ===`
+| Parameter                      | Explanation                                                     | Input type | Default                                                                         |
+| ------------------------------ | --------------------------------------------------------------- | ---------- | ------------------------------------------------------------------------------- |
+| `--output-chunk-filename`      | The output filename for additional chunks                       | string     | filename with [id] instead of [name] or [id] prefixed                           |
+| `--output-filename`            | The output filename of the bundle                               | string     | [name].js                                                                       |
+| `--output-jsonp-function`      | The name of the JSONP function used for chunk loading           | string     | webpackJsonp                                                                    |
+| `--output-library`             | Expose the exports of the entry point as library                | string     |
+| `--output-library-target`      | The type for exposing the exports of the entry point as library | string     | var                                                                             |
+| `--output-path`                | The output path for compilation assets                          | string     | Current directory                                                               |
+| `--output-pathinfo`            | Include a comment with the request for every dependency         | boolean    | false                                                                           |
+| `--output-public-path`         | The public path for the assets                                  | string     | /                                                                               |
+| `--output-source-map-filename` | The output filename for the SourceMap                           | string     | [name].map or [outputFilename].map                                              |
+| `--build-delimiter`            | Display custom text after build output                          | string     | Default string is null. You could provide a string such as `=== Build done ===` |
 
+### Cascade Options
+
+Sometime we want to add a option conditionally to an existing script, without writing writing the same script again so we can cascade option to it with just using `--` before the option and access it in our config file.
+
+For npm:
+
+```bash
+"build": "webpack --mode production --config webpack.prod.config.js"
+"build:compress": "npm run build -- --env.presets compress",
+```
+
+For yarn, we can skip `--` :
+
+```bash
+"build:compress": "yarn build --env.presets compress",
+```
+
```

Comment:
it lacks javascript example where you read the variable in config

Reply:
Great Suggestion. I will do it.

Reply:
You want to do it in this iteration right?